docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -68,6 +68,14 @@ You can also use the `--dev-start-rethinkdb` command line option to automaticall
 e.g. `bigchaindb --dev-start-rethinkdb start`. Note that this will also shutdown rethinkdb when the bigchaindb process stops.
 The option `--dev-allow-temp-keypair` will generate a keypair on the fly if no keypair is found, this is useful when you want to run a temporary instance of BigchainDB in a Docker container, for example.
 
+### Options
+The log level for the console can be set via the option `--log-level` or its
+abbreviation `-l`. Example:
```

Comment:
Maybe say that the allowed log levels are `DEBUG`, `INFO`, `WARNING`, `ERROR` and `CRITICAL` (maybe also `NOTSET`?) with a link to further explanation of what those log levels typically mean, e.g. https://docs.python.org/3.6/library/logging.html#levels